### DISCUSSION TITLE
Auditing of user sign-ins

### DISCUSSION THREAD

**Discussion by ElijaRock** [immich-app/immich#1615] [Feature Request]
### Feature detail

When a user logs in, their username and date they signed in is logged and displayed on the administration page. There should be an option to include other info like location of sign in. Maybe the admin can get an email notifying them when the server is being accessed from a new location.

### Platform

Server

**Comment by bo0tzz** [comment]
This seems more like something for an authentication server like Authentik/Keycloak, perhaps?
